chapters/en/chapter10/5.mdx

```diff
@@ -0,0 +1,62 @@
+# Use your annotated dataset
+
+We will learn now how to export and use the annotated data that we have in Argilla. 
+
+## Load the dataset
+
+First, we'll need to make sure that we're connected to our Argilla instance as in the previous steps:
+
+```python
+import argilla as rg
+
+HF_TOKEN = "..."  # only for private spaces
+
+client = rg.Argilla(
+    api_url="...",
+    api_key="...",
+    headers={"Authorization": f"Bearer {HF_TOKEN}"},  # only for private spaces
+)
+```
+
+And now, we'll load the dataset that we'll be working with:
+
+```python
+dataset = client.datasets(name="ag_news")
+```
+
+Loading the dataset and calling its records with `dataset.records` is enough to start using your dataset and records for your own purposes and pipelines. However, we'll also learn how to do a few optional operations, like filtering the records and exporting your dataset to the Hugging Face Hub.
+
+## Filter the dataset
+
+Sometimes you only want to use the records that have been completed, so we will first filter the records in our dataset based on their status:
+
+```python
+status_filter = rg.Query(filter=rg.Filter([("status", "==", "completed")]))
+
+filtered_records = dataset.records(status_filter)
+```
+
+>[!TIP]
+>⚠️ Note that the records with `completed` status (i.e., records that meet the minimum submitted responses configured in the task distribution settings) could have more than one response and that each response can have any status from `submitted`, `draft` or `discarded`.
+
+Learn more about querying and filtering records in the [Argilla docs](https://docs.argilla.io/latest/how_to_guides/query/).
+
+## Export to the Hub
```

Comment:
Might be good to link the exported dataset here, so that people can take a peek at it.

Reply:
Same here - we can put the link to the actual dataset here.